circle.yml

```diff
@@ -33,16 +33,9 @@ jobs:
           command: make test
       - run:
           name: SonarQube upload
-          command: make sonarqube
-      - run:
-          name: Update Transifex
           command: |
-            if [ "${CIRCLE_BRANCH}" == "${DEPLOY_BRANCH}" ]; then
-              pip install transifex-client
-              echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"${TRANSIFEX_USER}"$'\npassword = '"${TRANSIFEX_API_TOKEN}"$'\ntoken = '""$'\n' > ~/.transifexrc
-              tx push -s
```

Comment:
Instead of pushing explicitly, you can set up Transifex to autoupdate the resources [from the Web interface](https://www.transifex.com/mapbox/mapbox-java/content/). It updates at least daily based on the raw URLs you provide.

Reply:
Do we want to do this now or in the future once we actually have strings to localize? We currently have nothing to translate in 3.0.

Reply:
What’ll happen to [these resources](https://www.transifex.com/mapbox/mapbox-java/content/) in 3.0? Does 3.0 have hard-coded strings in the code, or are there no user-facing strings in the first place?

Reply:
no user-facing strings (besides of course the ones coming from APIs). There are no UI components in 3.0 now.